src/read.c

```diff
@@ -4677,6 +4677,42 @@ static avifResult avifMetaFindAlphaItem(avifMeta * meta,
     return AVIF_RESULT_OK;
 }
 
+// On success, this function returns AVIF_RESULT_OK and does the following:
+// * If a nclx property was found in |properties|:
```

Comment:
Document under what conditions this functions fails, and that the output parameters may be set when the function fails.

Reply:
Done.